src/main/java/org/entando/entando/plugins/jpseo/web/page/validator/SeoPageValidator.java

```diff
@@ -58,5 +63,28 @@ public boolean checkFriendlyCode(String pageCode, String friendlyCode) {
         return true;
     }
 
+    public boolean validateFriendlyCodeByLang(Map<String, SeoDataByLang> seoDataByLang, Errors errors) {
+
+        List friendlyCodes = new ArrayList<String>();
+        seoDataByLang.forEach(
+                (key, value) -> {
+                    if (null != value.getFriendlyCode()) {
```

Comment:
Ignore also empty friendly codes. By default app-builder sends empty friendly codes, so an error would be triggered for pages having no friendly codes defined.

Reply:
Done